test/test_multiS.jl

```diff
@@ -0,0 +1,88 @@
+using LinearAlgebra: length
+using ACE, ACEbase, Test, ACE.Testing
+using ACE: evaluate, SymmetricBasis, NaiveTotalDegree, PIBasis
+
+using StaticArrays
+
+
+##
+    
+# construct the 1p-basis
+D = NaiveTotalDegree()
+maxdeg = 6
+ord = 3
+
+B1p = ACE.Utils.RnYlm_1pbasis(; maxdeg=maxdeg, D = D)
+
+# generate a configuration
+nX = 54
+Xs() = ACE.State(rr = rand(SVector{3, Float64}), u = rand())
+cfg = ACEConfig([Xs() for i in 1:nX])
+a(x) = fieldnames(typeof(x))
+
+φ = ACE.Invariant()
+pibasis = PIBasis(B1p, ord, maxdeg; property = φ)
+basis = SymmetricBasis(pibasis, φ)
+       
+BB = evaluate(basis, cfg)
+
+c_m = rand(SVector{7,Float64}, length(BB))
+
+singlProp = [ACE.LinearACEModel(basis, rand(length(BB)), evaluator = :standard) for i in 1:length(c_m[1])]
+multiProp = ACE.LinearACEModel(basis, c_m, evaluator = :standard)
+
+@info("set_params!")
+c_s = [[c_m[j][i] for j in 1:length(c_m)] for i in 1:length(c_m[1])]
+
+ACE.set_params!(multiProp,c_m)
+for i in 1:length(c_m[1])
+    ACE.set_params!(singlProp[i],c_s[i])
+    print_tf(@test(c_s[i] ≈ singlProp[i].c))
+end
+println()
+
+@info("evaluate")
+
+for i in 1:length(c_m[1])
+    print_tf(@test(evaluate(singlProp[i],cfg).val ≈ evaluate(multiProp,cfg)[i].val))
+end
+println()
+
+
+@info("grad_params")
+#for now grad_params only gets B ONCE, we could copy it multiple times.
+for i in 1:length(c_m[1])
+    print_tf(@test(getproperty.(ACE.grad_params(singlProp[i],cfg),:val) ≈ getproperty.(ACE.grad_params(multiProp,cfg)[:,i],:val)))
+end
+println()
+
+
+@info("grad_config")
+
+function config_dist_eq(A,B)
+    pass = true
+    for i in 1:length(A)
+        if(A[i].rr != B[i].rr)
```

Comment:
should you use `!( A[i].rr ≈ B[i].rr )`?

also - this seems to me like you write it as 
```julia
config_dist_eq(A,B) = all( a.rr ≈ b.rr for (a, b) in zip(A, B) )
```
And in fact, I would argue the "correct" way to implement this is simply `A ≈ B` and then overload `isapproc` to provide an `isapprox` for `State` and `DState` (but maybe that's something you don't need to worry about now - though worth filing an issue for it if you'd be willing)